paths/paths_bsd.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build freebsd openbsd
```

Comment:
generally having separate files like this is overkill unless you need to vary your imports (e.g. like I need to avoid x/sys/unix import on Windows)

I'd just make this a little func in paths_unix.go looking at runtime.GOOS which'll get DCE+inlined away to nothing.

Reply:
Good call, done.